components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.common/src/main/java/org/wso2/carbon/identity/api/server/application/management/common/ApplicationManagementServiceHolder.java

```diff
@@ -36,53 +40,70 @@
 
     private static OAuthServerConfiguration oAuthServerConfiguration;
 
-    public static ApplicationManagementService getApplicationManagementService() {
+    private static TemplateManager templateManager;
+
```

Comment:
make the constructor of ApplicationManagementServiceHolder private

Reply:
Added with 61c58f9c3daf7bc9ce32fe5c8c63fada52fda397